pkg/backend/display/diff.go

```diff
@@ -196,7 +196,7 @@ func renderSummaryEvent(action apitype.UpdateKind, event engine.SummaryEventPayl
 
 				// Print a summary of the changes of this kind
 				fprintIgnoreError(out, opts.Color.Colorize(
-					fmt.Sprintf("    %s%d %s%s%s\n", op.Prefix(), c, planTo, opDescription, colors.Reset)))
+					fmt.Sprintf("    %s%d %s%s%s\n", op.Prefix(true /* done color */), c, planTo, opDescription, colors.Reset)))
```

Comment:
Nit: [our pattern](https://github.com/pulumi/pulumi/blob/master/pkg/codegen/python/gen.go#L571) is to use the function parameter name with no spaces.
```suggestion
					fmt.Sprintf("    %s%d %s%s%s\n", op.Prefix(true /*done*/), c, planTo, opDescription, colors.Reset)))
```